client/extensions/woocommerce/app/products/product-create.js

```diff
@@ -61,7 +62,8 @@ class ProductCreate extends React.Component {
 	}
 
 	onSave = () => {
-		// TODO: Add action dispatch to save this product.
+		const { siteId, product } = this.props;
```

Comment:
does this component need to know site id? Shouldn't create product be able to get siteId or at least have a default.  @DanReyLop and I discussed here: https://github.com/Automattic/wp-calypso/pull/14438#discussion_r118713391